plugin/ansible_vsphere_gosv_log.py

```diff
@@ -727,15 +728,21 @@ def _print_test_results(self):
             self._display.display(msg, color=C.COLOR_VERBOSE)
             return
 
-        # Block test cases when deploy_vm failed
-        deploy_vm_failed = False
+        # Block test cases when deploy_vm or installing tools failed
+        testcase_blocked = False
+        if self._play_name == 'env_setup':
+            # Test cases are blocked by env_setup failure
+            testcase_blocked = True
+
         for test_id in self.test_runs:
-            if 'deploy_vm' in self.test_runs[test_id].name:
-                deploy_vm_failed = (self.test_runs[test_id].status == 'Failed')
+            blocker_pattern = r'deploy_vm|ovt_verify_.*_install|wintools_complete_install_verify'
```

Comment:
This line `blocker_pattern = ` can be put outside the loop.

Reply:
Yes. Moved it out. Thanks!